src/membraneWebRTC.ts

```diff
@@ -198,12 +200,26 @@ export interface Callbacks {
    * @param {string} trackId - id of track that changed encoding
    * @param {TrackEncoding} encoding - new encoding
    */
-  onTrackEncodingChanged?: (peerId: string, trackId: string, encoding: TrackEncoding) => void;
+  onTrackEncodingChanged?: (
+    peerId: string,
+    trackId: string,
+    encoding: TrackEncoding
+  ) => void;
 
   /**
    * Called every time a local peer is removed by the server.
    */
   onRemoved?: (reason: string) => void;
+
+  /**
+   * Called every time an update about voice activity is received from the server
+   * `vad_status` has two possible values: "silence" and "speech"
+   */
+  onVadNotification?: (
+    peerId: string,
+    trackId: string,
+    vadStatus: "speech" | "silence"
```

Comment:
I would call this `onVoiceActivityChanged` or `onTrackVoiceActivityChanged` (we have similar event in the RTC Engine and it fits better `onTrackEncodingChanged`) . 

What about introducing a separate type for `vadStatus` similar to TrackEncoding? If we decide to do this, please ensure it is correctly displayed in the documentation